### PR TITLE
New endpoint for user connect.

### PR DESCRIPTION
**Pull request by melvinsoft**
This is a new endpoint that allows to set a new password to an existing user.

### CODE REVIEW

**Comment by melvinsoft** [comment]
[(3) Penn State: create new 'user connect' endpoint, push to staging and production](https://trello.com/c/YnfE3Cj5/1324-3-penn-state-create-new-user-connect-endpoint-push-to-staging-and-production)

File: lms/djangoapps/appsembler_api/views.py [hunk 1]
```diff
@@ -164,6 +164,56 @@ def post(self, request):
...
+        new_password = data.get('password', '')
+
+        try:
+            user = User.objects.get(email=email)
+
+            if not new_password.strip():
+                raise ValidationError('Password is empty')
+
+            user.set_password(new_password)
+            print user.password
```
Comment (tkeemon) [inline lms/djangoapps/appsembler_api/views.py]: You should probably remove this before merging this PR.

**Comment by melvinsoft** [comment]
@tkeemon Done, print removed. Thanks!

**Comment by johnbaldwin** [comment]
@melvinsoft Looks Great!

**Pull request merged**